frontend/components/GraveContents.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+import React, { useContext } from 'react';
+import { Box, Text, Flex, Icon } from '@chakra-ui/react';
+import { FaCog } from 'react-icons/fa';
+import GravePageAssets from '@/components/GravePageAssets';
+import ShareButton from '@/components/ShareButton';
+import { WalletContext } from '@/components/wallet/WalletContext';
+import { formatAddress } from '@/utils/tokenUtils';
+import Link from 'next/link';
+
+export default function GraveContents({
+  account = null,
+}: {
+  account?: string | null;
+}) {
+  const walletContext = useContext(WalletContext);
+  const { account: connectedAccount, graveVault } = walletContext;
+  let graveTitle = 'YOUR GRAVEYARD';
+  let pageGraveVault = null;
+  let pageAccount = null;
+  // account not present because on manage page
+  // account present and same as connected account
+  // account present but not same as connected account
+  // no connected account
```

Comment:
what's this?

Reply:
the possible user states that this component handles